src/models/Call.ts

```diff
@@ -566,18 +574,77 @@ export class ElementCall extends Call {
         this.checkDestroy();
     }
 
-    private static generateWidgetUrl(client: MatrixClient, roomId: string): URL {
-        const baseUrl = window.location.href;
-        let url = new URL("./widgets/element-call/index.html#", baseUrl); // this strips hash fragment from baseUrl
+    private static appendCallNotifIntent(params: URLSearchParams, client: MatrixClient, roomId: string): void {
```

Comment:
rename to what does the notif here mean?
I think just using the word `callIntent` or `appendIntentParam` would make more sense?
Notif might be misunderstood as setting `sendNotificationType` url param